JMMServer/Repositories/BaseCachedRepository.cs

```diff
@@ -115,7 +126,7 @@ public virtual void Delete(T cr)
                 EndDeleteCallback?.Invoke(cr);
             }
         }
-        public virtual void Delete(List<T> objs)
+        public virtual void Delete(IReadOnlyCollection<T> objs)
```

Comment:
You're right it could have been IEnumerable. I chose IReadOnlyCollection because it was the lowest interface that had a Count property which the current implementation was using (as you mentioned).